SKILLS.md

```
@@ -0,0 +1,71 @@
+
+### Confirm driver matches the image you pulled
+
+```bash
+nvidia-smi --query-gpu=driver_version --format=csv,noheader | head -1
+# Cross-check against the Container / driver matrix in docs/backends/trtllm/README.md
+```
+
+## Adding skills
+
+1. Add a markdown file under `.claude/skills/<skill-name>/SKILL.md` with frontmatter:
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)